### PR TITLE
add Euclidean plane

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
Realized we were missing this obvious counterexample once https://topology.pi-base.org/spaces?q=completely%20normal%20%2B%20t_2%20%2B%20~lots was posted to https://math.stackexchange.com/questions/4683906/

**Comment by StevenClontz** [comment]
This needs to have references added.

**Comment by prabau** [comment]
Real line (S25): should we remove P36 (connected), as it's derivable from P38 (arc connected)?

**Comment by StevenClontz** [comment]
Note that my reference on mathSE was closed for being too broad. 😞

**Comment by prabau** [comment]
Referencing just that one mathse post did seem somewhat circular.  The space was mentioned in Counterexamples, number 28.9. Maybe for the plane we could just briefly give justifications in pi-base itself? (most being because a product of spaces satisfying the same property, which in the long run will be deduced automatically).  Let me know if you want me to help out.

The only non obvious property is why it's not a LOTS.  (removing a point does not disconnect the space, but easier to find a ref for it)

**Comment by prabau** [comment]
"has a cut point" ?

**Comment by StevenClontz** [comment]
> every LOTS with at least three points has a cut point.

lol the first thing I thought of when you suggested the necessary cut point property for LOTS: "ah geez, now we have to have a 'has at least three points' property"

**Comment by prabau** [comment]
I don't think we need to introduce cut point as a property right now.  But we can still mention it in the justification.  If people don't know what that it, they can just look it up (it's defined both in Willard and wikipedia).

**Comment by StevenClontz** [comment]
Yes -  I don't want to introduce "has at least three points" until the point where we can qualify generic properties with a cardinal.

File: spaces/S000176/properties/P000133.md
Comment (prabau) [inline spaces/S000176/properties/P000133.md:12]: I think we need to say "Every connected {P133} space ...", since talking about cut points assumes the space is connected to start with (which the plane is).

**Review by prabau — APPROVED** [review]
Approving, modulo the comment above.

**Comment by StevenClontz** [comment]
Added "connected" quaifier. Though I wonder if it will be cleaner ultimately to define a "cut point" `p` in pi-Base as a point of *any* space for which `X\setminus\{p\}` is disconnected, whether or not `X` is connected to begin with.

**Comment by prabau** [comment]
Side remark to myself: a connected LOTS is either a singleton, or uncountable (due to T81, which apparently is a non-trivial fact due to Urysohn (have not checked the proof myself).  Maybe there is a more direct proof for LOTS.  Anyway, that shows that if we decide to introduce the "has cut point" property in the future, we would not need to introduce the property "has at least three point" or a generic version of it.

**Comment by prabau** [comment]
> 



> Added "connected" quaifier. Though I wonder if it will be cleaner ultimately to define a "cut point" `p` in pi-Base as a point of _any_ space for which `X\setminus\{p\}` is disconnected, whether or not `X` is connected to begin with.

Yeah, not sure what would be best.  That could be a topic to talk about in our next zoom meeting.

**Comment by prabau** [comment]
@GeoffreySangston I think adding the property "Has a cut point" (or is it "cut-point"?) would be fine.  It's a common notion, with relationships to other existing pi-base properties already.  Would maybe allow to streamline or remove some redundant traits.

Not sure myself about "strong local homogeneity".  Maybe we can start with "has a cut point" and after that open an issue to discuss the other property?  I.e., prevalence in the literature, implications, etc.

Note that theorem 6 in the mentioned paper required a "Has a non-cut point".  That's different.  Do we need to introduce that too?  Maybe hold off on that one for now?

@StevenClontz Comments?

**Comment by danflapjax** [comment]
> Note that theorem 6 in the mentioned paper required a "Has a non-cut point". That's different. Do we need to introduce that too? Maybe hold off on that one for now?

The complement of this would be (excluding some finite cardinalities) describing a cut point space, which is an established idea.

**Comment by GeoffreySangston** [comment]
Since 'Cut point space' has a lot more theorems, it seems, than 'Has a cut point', I'll not do that in the PR for 'Has a cut point'. But I do think it's a good concept to add and should perhaps also be discussed in an issue if somebody wants to do that.

**Pull request merged**
